data-access-layer/eventmesh/src/ApiServerClient.js

```diff
@@ -1033,15 +1034,15 @@ class ApiServerClient {
     const resourceType = crd.kind === 'SFService' ? CONST.APISERVER.RESOURCE_TYPES.INTEROPERATOR_SERVICES : CONST.APISERVER.RESOURCE_TYPES.INTEROPERATOR_PLANS;
     return Promise.try(() => apiserver
       .apis[CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR][CONST.APISERVER.API_VERSION]
-      .namespaces(CONST.APISERVER.DEFAULT_NAMESPACE)[resourceType].post({
+      .namespaces(CONST.APISERVER.DEFAULT_NAMESPACE)[resourceType].post({ // Default NS is used in this context since it is only done for the BOSH usecase, shouldn;t be used in any other context.
```

Comment:
Same query as above

Reply:
same here, these utility functions are used in BOSH based deployment to create SFplans and SFServices from the catalog, which happens separately in the K8s based deployments. Hence I didn't disturb these and kept them as is, added comment so that it is not used elsewhere.